### ISSUE TITLE
Don't write data to test directories

### ISSUE THREAD

**Issue by mivade** [pennmem/ram_utils#232]
For whatever reason, tests end up writing a lot of data into the `ramutils.test` package. This is bad. We should instead be using some specific directory for this so that we don't accidentally include package data in built packages.

**Comment by leondavis1** [comment]
Do you know what tests actually write this data?

**Comment by mivade** [comment]
No. I think we just have to run it to see what files are produced and figure out from there where it's happening.

**Comment by leondavis1** [comment]
Found it; it's `test.tasks.test_odin`
